### PR TITLE
Create ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57991](https://cumulocity.atlassian.net/browse/MTM-57991)
Original PR: [5834](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5834)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix: Add pipe to generate asset/group/device map marker pop… (#5771) [GRAFT][release/cd] (#5834)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add map marker popups for assets, groups, and devices
```

File: content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57991
+version: 1019.9.1
+---
+fix: Add pipe to generate asset/group/device map marker pop… (#5771) [GRAFT][release/cd] (#5834)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md:17]: The Claude AI suggested to use the following body:
```suggestion
Added a pipe to generate map marker popups for assets, groups, and devices on maps. The popups display identifying information about the asset, group, or device.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed map marker popups link to assets and devices
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md:3]:
```suggestion
title: Map marker popup correctly links to assets and devices
```

File: content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57991
+version: 1019.9.1
+---
+The marker popup now referencing correctly to an asset or a group if the link in the popup is clicked.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-9-1-fix-add-pipe-to-generate-asset-group-device-map-marker-pop.md:17]:
```suggestion
The link in the marker popup of the map now correctly references the respective asset or a group.
```

**Pull request merged**